components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -3210,6 +3210,7 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
     async listBilledUsage(
         ctx: TraceContext,
         attributionId: string,
+        startedTimeOrder: number,
```

Comment:
I think the type here can also be updated to use `SortOrder`.

Reply:
I wrote a comment [here](https://github.com/gitpod-io/gitpod/pull/11694#discussion_r934211465) that after I changed it to `SortOrder`, TS complained about expecting a different type.